src/views/states/list/hooks/useSortStates.ts

```diff
@@ -0,0 +1,40 @@
+import { useState } from 'react';
+
+import { ThSortType } from '@patternfly/react-table/dist/esm/components/Table/base';
+import { V1beta1NodeNetworkState } from '@types';
+
+const useSortStates = (
+  nodeNetworkStates: V1beta1NodeNetworkState[],
+): { sortedStates: V1beta1NodeNetworkState[]; nameSortParams: ThSortType } => {
+  const [activeSortIndex, setActiveSortIndex] = useState<number | null>(null);
+  const [activeSortDirection, setActiveSortDirection] = useState<'asc' | 'desc' | null>(null);
+
+  let sortedStates = nodeNetworkStates;
+  if (activeSortIndex === 1) {
+    sortedStates = nodeNetworkStates.sort((a, b) => {
+      const aValue = a.metadata.name;
+      const bValue = b.metadata.name;
+
+      if (activeSortDirection === 'asc') {
+        return (aValue as string).localeCompare(bValue as string);
+      }
+      return (bValue as string).localeCompare(aValue as string);
```

Comment:
Just wondering if we couldn't use `as string` a bit earlier to avoid repeating ourselves so much in these lines..
Maybe if we better used
```
const aValue = a.metadata.name as string;
const bValue = b.metadata.name as string;
```
 ?

Reply:
Yes sure